collection_functions.php

```diff
@@ -0,0 +1,103 @@
+<?php
+
+
+function getCampaignNames($db)
+{
+    $campaigns = $db->prepare("SELECT DISTINCT `cycle` FROM `scenarios`;");
+    $campaigns->setFetchMode(PDO::FETCH_ASSOC);
+    $campaigns->execute();
+    $cycleArray = array();
+    $results = $campaigns->fetchAll();
+    foreach ($results as $cycleName) {
+        array_push($cycleArray, $cycleName["cycle"]);
+    }
+    return $cycleArray;
+}
+
+function getScenarios($db)
+{
+    $queryString = "SELECT * FROM `scenarios` ORDER BY `position`;";
+    $query = $db->prepare($queryString);
+    $query->setFetchMode(PDO::FETCH_ASSOC);
+    $query->execute();
+    $bigArray = $query->fetchAll();
+    return $bigArray;
+}
+
+function scenarioToNotOwn($db,$toNotOwn)
+{
+    $insertToNotOwn = $db->prepare("UPDATE scenarios SET owned = 0 WHERE `name` =:toNotOwn;");
+    $insertToNotOwn -> bindParam(':toNotOwn', $toNotOwn);
```

Comment:
dont want spaces either side of -> (for rest of this file too)

Reply:
fixed